docling/datamodel/pipeline_options.py

```diff
@@ -226,15 +226,15 @@ class PictureDescriptionApiOptions(PictureDescriptionBaseOptions):
     params: Dict[str, Any] = {}
     timeout: float = 20
 
-    prompt: str = "Describe this image in a few sentences."
+    prompt: str = "Describe this image in a few sentences. Extract semantic components from the image (objects, actions, colors, etc.) and generate a star-structure graph with the image as the central node. Link surrounding nodes with labeled relationships to form a unique ontology-style concept map and add this to the description of the image."
```

Comment:
This looks like a very specific prompt which fits your special use case. Please keep the generic one or improve it in a generic way.